internal/types/application_version.go

```diff
@@ -15,6 +15,7 @@ type ApplicationVersion struct {
 	CreatedAt     time.Time      `db:"created_at" json:"createdAt"`
 	ArchivedAt    *time.Time     `db:"archived_at" json:"archivedAt,omitempty"`
 	Name          string         `db:"name" json:"name"`
+	Link          string         `db:"link" json:"link"`
```

Comment:
I think `LinkTemplate` would be a better name, also in the DB. Then in the deployment we can keep `ApplicationLink` and `ApplicationLinkTemplate` separate and it is clear when something is a template and when it isn't. Also, we can then only expose the already executed `ApplicationLink` via JSON.